crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -323,6 +490,27 @@ pub(crate) fn check_docstring(
         visitor.finish()
     };
 
+    // DOC201
+    if checker.enabled(Rule::DocstringMissingReturns) {
+        if !is_property(definition, checker) && docstring_sections.returns.is_none() {
+            for body_return in &body_entries.returns {
+                let diagnostic = Diagnostic::new(DocstringMissingReturns, body_return.range());
+                diagnostics.push(diagnostic);
+            }
+        }
```

Comment:
Does emitting a diagnostic for each return statement match the behavior of the upstream rule? It feels a bit verbose to me. What do you think, would it be enough to just flag the first return statement?